src/i18n/ru/potato.js

```diff
@@ -0,0 +1,32 @@
+export default {
+  potato: {
+    1: 'Henry Spalding первым посадил картофель в Айдахо в 1837 году',
+    2: 'Картофель фри появился в Америке, когда Thomas Jefferson подал его на ужин в Белом доме.',
+    3: 'Любители картофеля в Америке съели более 4 миллионов тонн картофеля фри в различных его формах и размерах в 2017 году.',
+    4: 'Картофель - это мощный афродизиак, как говорят врачи в Ирландии.',
+    5: 'Среднестатистический американец съедает 140 фунтов картофеля в год. Немцы съедают более 200 фунтов за год.',
+    6: 'Самый большой выращенный плод картофеля весил 18 фунтов и 4 унции согласно Книге рекордов Гиннесса. Он был выращен в Англии в 1795 году.',
+    7: 'Картофель является четвертым по популярности продуктом питания в мире – после пшеницы, кукурузы и риса.',
+    8: 'Картофель выращивают более чем в 125 странах мира.',
+    9: 'Каждый год в мире вырастает столько картофеля, что его достаточно, чтобы покрыть четырехполосную автомагистраль вокруг земного шара шесть раз.',
+    10: 'Китай производит больше всего картофеля в мире.',
+    11: 'Картофель был первым овощем, выращенным в космосе.',
+    12: 'Картофель совсем не содержит глютена.',
+    13: 'Одной из основных единиц измерения времени у Инков было время, необходимое для приготовления картофеля.',
+    14: 'Если картофель подвергается воздействию солнечных лучей во время созревания, он становится зеленым и ядовитым.',
+    15: 'Необходимо 10,000 фунтов картофеля, чтобы приготовить 2,500 фунтов картофельных чипсов.',
```

Comment:
The comma is a decimal separator in Russia, it would be better to remove commas or change them for point or space.